pkg/apply/task/inv_add_task_test.go

```diff
@@ -167,8 +168,8 @@ func TestInvAddTask(t *testing.T) {
 			if result.Err != nil {
 				t.Errorf("unexpected error running InvAddTask: %s", result.Err)
 			}
-			actual, _ := client.GetClusterObjs(t.Context(), nil)
-			if !tc.expectedObjs.Equal(actual) {
+			actual, _ := client.Get(t.Context(), nil, inventory.GetOptions{})
```

Comment:
Not super clear what this is doing.
Why ignore the error?
Why take a nil?

Reply:
Gotcha. 

nit: Can you just pass in something believable with a comment that it doesn't matter? It would just make it a bit easier to read.